h2o-algos/src/main/java/hex/generic/GenericModel.java

```diff
@@ -224,6 +256,63 @@ private void predict(Chunk[] cs, NewChunk[] ncs) throws PredictException {
         }
     }
 
+    private class GenericScoreContributionsSortingTask extends MRTask<GenericScoreContributionsSortingTask> {
+        private final int _topN;
+        private final int _bottomN;
+        private final boolean _compareAbs;
+        private PredictContributions predictContributions;
```

Comment:
@michalkurka We can use PredictContributions interface instead of the whole wrapper.